### PR TITLE
Update unit tests

### PR DESCRIPTION
**Pull request by bsweeney**
- move fonts to tests resources directory
- remove Intel Clear-Light
- add W3C unit test font Ahem
- add new font info test

### CODE REVIEW

File: tests/FontLib/FontTest.php [hunk 1]
```diff
@@ -22,16 +22,30 @@ public function testLoadFileNotFound()
...
+        $trueTypeFont = Font::load('tests/resources/fonts/ahem/ahem.ttf');
 
         $this->assertInstanceOf('FontLib\TrueType\File', $trueTypeFont);
     }
 
-    public function test12CmapFormat()
+    public function testGetFontInfoTTF()
     {
-        $trueTypeFont = Font::load('sample-fonts/NotoSansShavian-Regular.ttf');
+        $font = Font::load('tests/resources//fonts/ahem/ahem.ttf');
```
Comment (williamdes) [inline tests/FontLib/FontTest.php:34]:
```suggestion
        $font = Font::load('tests/resources/fonts/ahem/ahem.ttf');
```

**Review by williamdes — APPROVED** [review]
Looks nice !

**Comment by williamdes** [comment]
This backports my fix in some way: https://sources.debian.org/src/php-font-lib/0.5.4+dfsg-3/debian/patches/0002-Update-tests-to-add-more-assertions-and-use-fonts-no.patch/

I am quite glad that I can drop it in Debian

**Comment by williamdes** [comment]
Could you also apply the other ones ?
https://udd.debian.org/patches.cgi?src=php-font-lib&version=0.5.4%2Bdfsg-3

**Comment by bsweeney** [comment]
Is there anything else to do?
- `0001-lintian-national-encoding.patch` is no longer necessary since the autoloader was removed.
- `0003-Adapt-for-phpunit-10.patch` also not necessary? At least, the one change that _was_ required (removing the syntaxCheck configuration attribute) was already addressed in PR #125.

**Comment by williamdes** [comment]
Indeed, seems like everything is handled 🎉 Nothing more to do then.

Should SECURITY.md be gitattributes removed ?

**Comment by bsweeney** [comment]
I don't feel strongly either way on that one. We do remove it from Dompdf, though so I guess for the sake of consistency...

**Review by williamdes — APPROVED** [review]

**Pull request merged**